src/applications/gi/components/vet-tec/VetTecHeadingSummary.jsx

```diff
@@ -3,23 +3,39 @@ import React from 'react';
 
 import VetTecAdditionalResources from './VetTecAdditionalResources';
 
-const IconWithInfo = ({ icon, children }) => (
-  <p className="icon-with-info">
-    <i className={`fa fa-${icon}`} />
-    &nbsp;
-    {children}
-  </p>
-);
+const IconWithInfo = ({ icon, iconClassName, children, present }) => {
+  if (!present) return null;
+  return (
+    <p className="icon-with-info">
+      <i className={`fa fa-${icon} ${iconClassName}`} />
+      {children}
+    </p>
+  );
+};
 
 export const VetTecHeadingSummary = ({ institution }) => (
   <div className="heading row">
     <div className="usa-width-two-thirds medium-8 small-12 column">
       <h1>{institution.name}</h1>
       <div>
         <div className="usa-width-one-half medium-6 small-12 column">
+          <IconWithInfo
+            icon="star"
+            iconClassName="vads-u-color--gold"
+            present={
+              institution.preferredProvider &&
+              institution.preferredProvider === true
+            }
+          >
+            Preferred provider (<a>Learn more</a>)
+          </IconWithInfo>
           {institution.city &&
             institution.country && (
-              <IconWithInfo icon="map-marker">
+              <IconWithInfo
+                icon="map-marker"
+                present={institution.city && institution.country}
+              >
+                &nbsp;
```

Comment:
Why was this space introduced here?

Reply:
It added space between the icon and text. Before there was no space making the line look crowded.